src/connectorgen/registry-config.yaml

```diff
@@ -0,0 +1,27 @@
+# Only repositories on the allow list will be added to the registry.
+# The allow list can either contain a string that matches the repository name
+# or an organization name with a wildcard (*) as the repository name.
+# The comparison will be case-insensitive.
+allow:
+  # Allow Meroxa and Conduit organizations
+  - ConduitIO/*
+  - conduitio-labs/*
+  - meroxa/*
+
+  # Individual repositories
+  - derElektrobesen/conduit-connector-tarantool
+  - lovromazgon/conduit-connector-kafka-broker
+  - nickchomey/conduit-connector-surrealdb
+
+# The deny list is used to exclude specific repositories from the allow list.
+# The deny list takes precedence over the allow list.
+# The deny list can either contain a string that matches the repository name
+# or an organization name with a wildcard (*) as the repository name.
+# The comparison will be case-insensitive.
+deny:
+  # Deny specific repositories that are allowed in the orgs above
```

Comment:
Could we, instead of individually denying specific repositories, maybe only allow those that follow the format:

  - ConduitIO/conduit-connector-*
  - conduitio-labs/conduit-connector-*
  - meroxa/conduit-connector-*

This way, we wouldn't need to worry about manually updating this list on any other repository we might (we will) create in the future on any of these orgs.

Reply:
Good idea! I'll add the change 👍

Reply:
See https://github.com/ConduitIO/conduit-site/pull/270/commits/debb06d465c86599d070a758f794b6b7913a484d 👍